### PR TITLE
Formatting and doc fixes

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
Adds missing documentation and reorder the functions

### CODE REVIEW

File: libdrizzle/drizzle_local.h [hunk 1]
```diff
@@ -69,11 +69,52 @@ void drizzle_set_error(drizzle_st *con, const char *function, const char *format
...
+/*
+ * Initializing the shared ssl object, by calling pthread_once
+ *
+ * @params[in] initialized drizzle connection object
+ * @return true if the call to pthread_once was successful, false otherwise
+ */
 bool drizzle_library_init(drizzle_st*);
 
 /**
+ * Initialize a result structure.
```
Comment (ben-palmer-sociomantic) [inline libdrizzle/drizzle_local.h]: Missing params and return value.

**Review by ben-palmer-sociomantic — COMMENTED** [review]
One small comment, then LGTM.

File: libdrizzle-5.1/drizzle_client.h [hunk 2]
```diff
@@ -91,10 +91,10 @@ extern "C" {
...
  * @param[in] options Drizzle connection options to add.
- * @return Same return as drizzle_create().
+ * @return newly allocated and configured connection object
  */
 DRIZZLE_API
-  drizzle_st *drizzle_create(const char *host, in_port_t port,
+drizzle_st *drizzle_create(const char *host, in_port_t port,
                              const char *user, const char *password,
                              const char *db,
                              drizzle_options_st *options);
```
Comment (daniel-zullo) [inline libdrizzle-5.1/drizzle_client.h]: You might also want to fix indentation for these lines

File: libdrizzle/drizzle_local.h [hunk 3]
```diff
@@ -69,11 +69,52 @@ void drizzle_set_error(drizzle_st *con, const char *function, const char *format
...
+/* Create a copy of drizzle_st instance
+ *
+ * @params[out] drizzle the target instance
+ * @params[in] from the instance to clone
+ * @return a newly allocated and configured copy of a drizzle connection
+ */
 drizzle_st *drizzle_clone(drizzle_st *drizzle, const drizzle_st *from);
 
+/*
+ * Initializing the shared ssl object, by calling pthread_once
```
Comment (daniel-zullo) [inline libdrizzle/drizzle_local.h]: Maybe `Initialize` to be consistent with the other functions/methods documentation?

**Review by daniel-zullo — COMMENTED** [review]
A couple of optional cosmetics comments and LGTM

**Comment by andreas-bok-sociomantic** [comment]
Updated based on comments from ben and dani 
```diff
diff --git a/libdrizzle-5.1/drizzle_client.h b/libdrizzle-5.1/drizzle_client.h
index e976b0c..6772679 100644
--- a/libdrizzle-5.1/drizzle_client.h
+++ b/libdrizzle-5.1/drizzle_client.h
@@ -95,9 +95,8 @@ extern "C" {
  */
 DRIZZLE_API
 drizzle_st *drizzle_create(const char *host, in_port_t port,
-                             const char *user, const char *password,
-                             const char *db,
-                             drizzle_options_st *options);
+                           const char *user, const char *password,
+                           const char *db, drizzle_options_st *options);
 
 /** @} */
 
diff --git a/libdrizzle/drizzle_local.h b/libdrizzle/drizzle_local.h
index 48d2606..9366818 100644
--- a/libdrizzle/drizzle_local.h
+++ b/libdrizzle/drizzle_local.h
@@ -78,7 +78,7 @@ void drizzle_free(drizzle_st *con);
 drizzle_st *drizzle_clone(drizzle_st *drizzle, const drizzle_st *from);
 
 /*
- * Initializing the shared ssl object, by calling pthread_once
+ * Initialize the shared ssl object, by calling pthread_once
  *
  * @params[in] initialized drizzle connection object
  * @return true if the call to pthread_once was successful, false otherwise
@@ -86,7 +86,10 @@ drizzle_st *drizzle_clone(drizzle_st *drizzle, const drizzle_st *from);
 bool drizzle_library_init(drizzle_st*);
 
 /**
- * Initialize a result structure.
+ * Initialize a result struct and add it to the connection object's result list
+ *
+ * @param[in] con initialized drizzle connection object
+ * @return a newly allocated result object
  */
 drizzle_result_st *drizzle_result_create(drizzle_st *con);
 

```

**Comment by ben-palmer-sociomantic** [comment]
LGTM

**Pull request merged**